lib/pb/protocol.proto

```diff
@@ -45,12 +45,13 @@ message PubAck {
 // Msg struct. Sequence is assigned for global ordering by
 // the cluster after the publisher has been acknowledged.
 message MsgProto {
-  uint64 sequence    = 1;  // globally ordered sequence number for the subject's channel
-  string subject     = 2;  // subject
-  string reply       = 3;  // optional reply
-  bytes  data        = 4;  // payload
-  int64  timestamp   = 5;  // received timestamp
-  bool   redelivered = 6;  // Flag specifying if the message is being redelivered
+  uint64 sequence        = 1;  // globally ordered sequence number for the subject's channel
+  string subject         = 2;  // subject
+  string reply           = 3;  // optional reply
+  bytes  data            = 4;  // payload
+  int64  timestamp       = 5;  // received timestamp
+  bool   redelivered     = 6;  // Flag specifying if the message is being redelivered
+  uint32 redelivery_count = 7;  // Number of times the message has been redelivered (count currently not persisted)
```

Comment:
I know it does not matter, but the field name in Go is `redeliveryCount`... and it would make it aligned with the other `= 1;`, `= 2;`, etc.. lines above :-)

Reply:
Not formatting the generated file, as the linter goes insane on the generated code. Did fix the proto file.